docs/integration/driver/display/fbdev.rst

```diff
@@ -39,3 +39,25 @@ node on the display (usually this is ``/dev/fb0``).
 If your screen stays black or only draws partially, you can try enabling direct rendering via ``LV_DISPLAY_RENDER_MODE_DIRECT``. Additionally,
 you can activate a force refresh mode with ``lv_linux_fbdev_set_force_refresh(true)``. This usually has a performance impact though and shouldn't
 be enabled unless really needed.
+
+
+Common mistakes
```

Comment:
Please check if this version is better for understanding:
```

When the Linux kernel initializes, it sets up subsystems like the framebuffer (fbdev) to manage display output. If an HDMI display is connected, the kernel detects it and allocates a portion of RAM as the framebuffer, which holds the pixel data for rendering images.

However, a common issue arises when the display is not powered on during the boot process. If the board is powered on before the screen, the kernel may incorrectly configure the framebuffer resolution. As a result, both the visible and virtual resolutions can be incorrect, leading to display problems.

This issue often occurs with HDMI connections where the display is powered up after the system has already booted.
```

Reply:
@EDGEMTech-GabrielC is on a holiday now, and will get back to this soon.